@here/harp-map-controls/lib/CameraAnimationBuilder.ts

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GeoCoordinates, MathUtils } from "@here/harp-geoutils";
+import { LookAtParams, MapView } from "@here/harp-mapview";
+import THREE = require("three");
+
+import { CameraKeyTrackAnimationOptions, ControlPoint } from "./CameraKeyTrackAnimation";
+
+/**
+ * A Utility class for the creation of Camera Animations
+ */
+export class CameraAnimationBuilder {
+    /**
+     * Get the current [[LookAtParams]] from [[MapView]]
+     *
+     * @param mapView The MapView
+     */
+    static getLookAtFromView(
+        mapView: MapView
+    ): Pick<LookAtParams, "target" | "tilt" | "heading" | "distance"> {
+        return {
+            target: mapView.target,
+            heading: mapView.heading,
+            tilt: mapView.tilt,
+            distance: mapView.targetDistance
+        };
+    }
+
+    /**
+     * Appends a [[ControlPoint]] to [[CameraKeyTrackAnimationOptions]]
+     *
+     * @param options The [[CameraKeyTrackAnimationOptions]] that the [[ControlPoint]] should be
+     * appended to
+     * @param point The [[ControlPoint]] to append.
+     * @param appendTime The time it should take from the former end of the animation to the
+     * appended [[ControlPoint]] in seconds, @default 10
+     */
+    static appendControlPoint(
```

Comment:
I don't see any usage of this function, can it be removed?

If not, please at least add a unit test to test it.